src/model/Model_Common_Include.i

```diff
@@ -72,7 +72,7 @@
   %init %{
     rb_eval_string("OpenStudio::IdfObject.class_eval { define_method(:to_" #_name ") { OpenStudio::Model::to" #_name "(self); } }");
     rb_eval_string("OpenStudio::Model::Model.class_eval { define_method(:get" #_name ") { |handle| OpenStudio::Model::get" #_name "(self, handle); } }");
-    rb_eval_string("OpenStudio::Model::Model.class_eval { define_method(:get" #_name "s) { OpenStudio::Model::get" #_name "s(self); } }");
+    rb_eval_string("OpenStudio::Model::Model.class_eval { define_method(:get" #_name "s) { | sorted = false | OpenStudio::Model::get" #_name "s(self, sorted); } }");
```

Comment:
Modify the ruby wrapper to provide getXXXXs(sorted=false), so that if you call getXXXs it sill does what it used to (unsorted for ConcreteModelObjects and ModelObjects alike)

(These code changes are very simple to see, but it took me a surprising amount of effort to get right)